pkg/apis/cluster/mutation/mutation_test.go

```diff
@@ -5,11 +5,50 @@ import (
 	"reflect"
 	"testing"
 
+	corev1 "k8s.io/api/core/v1"
+
```

Comment:
This blank maybe needs to be removed, otherwise, lint will report an error.

Reply:
ok